src/yang/ietf-qos-action.yang

```diff
@@ -475,45 +475,45 @@ module ietf-qos-action {
     leaf min-threshold-val {
       type uint64;
       description
-       "minimum value of red threshold";
+       "Minimum value of red threshold";
     }
     leaf min-threshold-unit {
       type identityref {
         base qos-types:red-threshold-unit;
       }
       description
-       "unit of minimum red threshold";
+       "Unit of minimum red threshold";
     }
     leaf max-threshold-val {
       type uint64;
       description
-       "maximum value of red threshold";
+       "Maximum value of red threshold";
     }
     leaf max-threshold-unit {
       type identityref {
         base qos-types:red-threshold-unit;
       }
       description
-       "unit of maximum red threshold";
+       "Unit of maximum red threshold";
     }
     leaf weight {
       type uint8;
       description
-       "the decay factor for the average queue size
+       "The decay factor for the average queue size
         calculation. the numbers are 2's exponent";
     }
     leaf max-probability-val {
       type uint64;
       description
-       "value of maximum probability value. this value need
+       "Value of maximum probability value. this value need
```

Comment:
s/ this/ This/

Also what does it mean it needs to be "interpreted". Interpreted how? Specify, that if the unit is probability-pc, then the value should be interpreted as a percentage of ..."